packages/web-components/fast-foundation/src/accordion/accordion.ts

```diff
@@ -111,18 +110,28 @@ export class FASTAccordion extends FASTElement {
             item.addEventListener("keydown", this.handleItemKeyDown);
             item.addEventListener("focus", this.handleItemFocus);
         });
+
         if (this.isSingleExpandMode()) {
-            const expandedItem: FASTAccordionItem | null =
-                this.findExpandedItem() ??
-                ((this._accordionItems[0] as unknown) as FASTAccordionItem);
-            expandedItem.setAttribute("aria-disabled", "true");
+            const expandedItem = this.findExpandedItem() as FASTAccordionItem;
```

Comment:
Does this type casting account for a situation where there are no accordion items?

Reply:
It doesn't - I think a pass can probably be done to guard against all this.